src/web/screens/home.tsx

```diff
@@ -64,7 +64,9 @@ async function getEmailContacts() {
 
 async function getApiKeysToRotate() {
   const apiKeys = await GetParticipantApiKeys();
-  return apiKeys.filter((apiKey) => apiKey.disabled === false && needsRotating(apiKey) === true);
+  return apiKeys.filter(
+    (apiKey) => apiKey.disabled === false && shouldRotateApiKey(apiKey) === true
```

Comment:
Per my other comments - perhaps best we move the `disabled` check inside `shouldRotateApiKey()`?